docs/src/architecture/npm-debug.log

```diff
@@ -0,0 +1,20 @@
+0 info it worked if it ends with ok
```

Comment:
Looks like this was checked in by mistake, should remove (may as well add `npm-debug.log` to `.gitignore` too, to avoid this in the future)